### PR TITLE
Update experiment status DB operation

### PR DESCRIPTION
**Pull request by hougangliu**
Part of #510

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/534)

### CODE REVIEW

File: pkg/controller/v1alpha2/experiment/util/api_util.go [hunk 1]
```diff
@@ -46,7 +48,18 @@ func DeleteExperimentInDB(instance *experimentsv1alpha2.Experiment) error {
...
 func UpdateExperimentStatusInDB(instance *experimentsv1alpha2.Experiment) error {
-
+	newStatus := &api_pb.ExperimentStatus{
+		StartTime:      convertTime2RFC3339(instance.Status.StartTime),
+		CompletionTime: convertTime2RFC3339(instance.Status.CompletionTime),
+		Condition:      getCondition(instance),
+	}
+	request := &api_pb.UpdateExperimentStatusRequest{
+		NewStatus:       newStatus,
+		ExperimentName: instance.Name,
```
Comment (gaocegege) [inline pkg/controller/v1alpha2/experiment/util/api_util.go]: format here is not gofmted, can you run gofmt for the file?

Comment (hougangliu) [reply pkg/controller/v1alpha2/experiment/util/api_util.go]: done! thanks

**Pull request merged**